riscv/riscv.mk.in

```diff
@@ -1300,6 +1300,28 @@ riscv_insn_ext_zacas = \
 	amocas_d \
 	$(if $(HAVE_INT128),amocas_q)
 
+riscv_insn_ext_zabha = \
+    amoadd_b \
+    amoand_b \
+    amomaxu_b \
+    amomax_b \
+    amominu_b \
+    amomin_b \
+    amoor_b \
+    amoswap_b \
+    amoxor_b \
+    amocas_b \
+    amoadd_h \
+    amoand_h \
+    amomaxu_h \
+    amomax_h \
+    amominu_h \
+    amomin_h \
+    amoor_h \
+    amoswap_h \
+    amoxor_h \
+    amocas_h
+
```

Comment:
Tabs, not spaces, in this file.  (Maybe one day we'll change that, but for now, let's stay consistent.)

Reply:
fixed.